src/construct-hub.ts

```diff
@@ -150,6 +151,17 @@ export interface ConstructHubProps {
    */
   readonly categories?: Category[];
 
+  /**
+   * Wire construct hub to use the failover storage buckets.
+   *
+   * Do not activate this property until you've populated your failover buckets
+   * with the necessary data.
+   *
+   * @see https://github.com/cdklabs/construct-hub/blob/dev/docs/operator-runbook.md#storage-disaster
+   * @default false
+   */
+  readonly failoverStorageActive?: boolean;
```

Comment:
The word "active" here is confusing (to me). I think this should be sufficient:

```suggestion
  readonly failoverStorage?: boolean;
```